radix-engine/tests/data_access/src/lib.rs

```diff
@@ -5,16 +5,30 @@ blueprint! {
     struct DataAccess {}
 
     impl DataAccess {
+        pub fn create_component_and_read_state() {
+            let component_address = Self {}.instantiate().globalize();
+            let address = DataAddress::Component(component_address, ComponentOffset::State);
+            let input = RadixEngineInput::ReadData(address);
+            call_engine(input)
+        }
```

Comment:
Can we move local component? Can we invoke a local component from another frame? Might be worth adding some tests :)